client/src/api/APISaveTest.res

```diff
@@ -1,3 +1,5 @@
+// WHAT is this
```

Comment:
It calls an API which used to save a canvas as JSON to use in integration tests.